velox/functions/sparksql/coverage/Coverage.cpp

```diff
@@ -31,7 +31,7 @@ int main(int argc, char** argv) {
   functions::sparksql::registerFunctions("");
 
   // Register Spark aggregate functions.
-  functions::sparksql::aggregates::registerAggregateFunctions("");
+  functions::sparksql::aggregate::registerAggregateFunctions("");
```

Comment:
This change makes this PR hard to review. Would it be possible to extract this refactor into a separate PR?

Reply:
Got it, I will revert and resubmit in another PR.